CHANGELOG.md

```diff
@@ -1,3 +1,172 @@
+# 55.3.0
+
+## Breaking changes
+
+## New deprecations
+
+## New features
```

Comment:
Flexible header has a new behavioral flag for opting in to better topLayoutGuide support. This is primarily useful when using the flexible header container view controller. To opt in to this new behavior, do the following:

```swift
let container = MDCFlexibleHeaderContainerViewController()
container.isTopLayoutGuideAdjustmentEnabled = true
```